R-package/R/lgb.Booster.R

```diff
@@ -49,7 +49,11 @@ Booster <- R6Class(
       }
       class(handle)     <- "lgb.Booster.handle"
       private$handle    <- handle
-      private$num_class <- 1L
+      if ("num_class" %in% names(params)){
+	private$num_class <- params$num_class
+      } else{
+	private$num_class <- 1L
+      }
       private$num_class <-
         lgb.call("LGBM_BoosterGetNumClasses_R", ret = private$num_class, private$handle)
```

Comment:
I am not very familiar with R-package details, but the logic should be same as python-package. All the parameters including num_class is set at [L26](https://github.com/Microsoft/LightGBM/pull/289/files/ed379175b0c5f34ff23d4313ffa9a55df1b878da#diff-acaf2c3176205deafe7683a35b929405R26) to C++. And R-package gets num_class from C++ here.